packages/advisor-components/src/common.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import compact from 'lodash/compact';
+import intersection from 'lodash/intersection';
+
+import { RuleContentRhel, TopicRhel } from './types';
+
+// eslint-disable-next-line no-undef
+export const barDividedList = (list: JSX.Element[]) =>
+  list.map((element, index) => (
+    <React.Fragment key={index}>
+      {element}
+      {index + 1 !== list.length && <strong className="verticalDivider">&nbsp;&#124;&nbsp;</strong>}
```

Comment:
I'd rather wrap the `{element}` in `span` and apply some styling to show table border on right.

Reply:
@karelhala, isn't it complicating things a bit? :) I mean, the result will be the same, however, the current &#124 vertical pipe already serves the punctuation function (comma). Writing CSS to repeat the Unicode character seems to me an extra thing :)

Reply:
If we are treating the pipe as comma I am fine with it.